src/homepageExperience/components/steps/Finish.tsx

```diff
@@ -152,7 +152,8 @@ export const Finish = (props: OwnProps) => {
           )}
         </FlexBox>
         {props.wizardEventName !== 'cliWizard' &&
-        props.wizardEventName !== 'arduinoWizard' ? (
+        props.wizardEventName !== 'arduinoWizard' &&
+        props.wizardEventName !== 'pythonSqlWizard' ? (
           <SampleAppCard
```

Comment:
For context - why are these excluded - because there isn't a sample app yet?

Reply:
Ya, we don't have a sample app for this SQL version yet.

Reply:
👍